app/templates/road-sign-concepts/road-sign-concept.hbs

```diff
@@ -1,78 +1,168 @@
+</AuToolbar>
+<AuToolbar @size="large">
+  <AuToolbarGroup>
+    <ul class="au-o-grid">
+      <li class="au-o-grid__item au-u-1-2 au-u-1-6@medium">
+        <img alt="" role="presentation" src={{@model.image}} />
+      </li>
+      <li class="au-o-grid__item au-u-1-2 au-u-1-6@medium">
+        <AuLabel>
+          {{t "roadSignConcept.attr.code"}}
+        </AuLabel>
+        <p>
+          {{@model.roadSignConceptCode}}
+        </p>
+      </li>
+      <li class="au-o-grid__item au-u-1-2 au-u-1-6@medium">
+        <AuLabel>
+          {{t "roadSignConcept.attr.categories"}}
+        </AuLabel>
+        {{#each @model.categories as |category|}}
+        <p>
+          {{category.label}}
+        </p>
+        {{/each}}
+      </li>
+      <li class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
+        <AuLabel>
+          {{t "roadSignConcept.attr.meaning"}}
+        </AuLabel>
+        <p>
+          {{@model.meaning}}
+        </p>
+      </li>
+    </ul>
+  </AuToolbarGroup>
+</AuToolbar>
 
-          <AuButton @alert={{true}} {{on "click" (fn (mut this.isOpen) true)}}>
-            {{t "roadSignConcept.crud.delete"}}
-          </AuButton>
-          <AuModal
-            @modalTitle={{t "utility.confirmation.title"}}
-            @modalOpen={{this.isOpen}}
-            @closeModal={{fn (mut this.isOpen) false}} as |Modal|
-          >
-            <Modal.Body>
-              <p>
-                {{t "utility.confirmation.body"}}
-              </p>
-            </Modal.Body>
-            <Modal.Footer>
-              <AuButton
-                @alert={{true}}
-                {{on "click" (fn this.removeRoadSignConcept @model)}}
-              >
-                {{t "roadSignConcept.crud.delete"}}
-              </AuButton>
-            </Modal.Footer>
-          </AuModal>
-        </c.footer>
-      </AuCard>
-    </div>
-  </div>
-</div>
\ No newline at end of file
+<AuDataTable @content={{this.model.subSigns}} @fields="Image Meaning Code"
+  @noDataMessage={{t "roadSignConcept.crud.noData" }} @size={{this.size}} @page={{this.page}} @sort={{this.sort}} as
+  |s|>
+  <s.menu as |menu|>
+    <menu.general>
+      <AuToolbar class="au-o-box">
+        <AuToolbarGroup>
+          <AuHeading @skin="2">
+            {{t "roadSignConcept.subSign.name" }}
+          </AuHeading>
+        </AuToolbarGroup>
+      </AuToolbar>
+    </menu.general>
+  </s.menu>
+  <s.content as |c|>
+    <c.header>
+      <th class="data-table__header-title">
+        {{t "roadSignConcept.attr.code" }}
+      </th>
+      <th class="data-table__header-title">
+        {{t "roadSignConcept.attr.imageHeader"}}
+      </th>
+      <th class="data-table__header-title">
+        {{t "roadSignConcept.attr.meaning"}}
+      </th>
+      <th class="data-table__header-title"></th>
+    </c.header>
+    <c.body as |subSign|>
+      <td>
+        {{subSign.roadSignConceptCode}}
+      </td>
+      <td>
+        <img src={{subSign.image}} alt="" role="presentation" />
```

Comment:
I think creating a `.road-sign` class which sizes the image would be a good idea. This class can replace [the hacky styles](https://github.com/lblod/frontend-mow-registry/blob/a2747baa81ffdecd3c2a3ca0a767e2cdc40d3364/app/styles/app.scss#L123) we added before.